src/tests/Microsoft.PowerFx.Connectors.Tests.Shared/Microsoft.PowerFx.Connectors.Tests.Shared.projitems

```diff
@@ -1,24 +1,269 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <MSBuildAllProjects Condition="'$(MSBuildVersion)' == '' Or '$(MSBuildVersion)' &lt; '16.0'">$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
-    <HasSharedItems>true</HasSharedItems>
-    <SharedGUID>b91ccc91-2a8d-4411-9a50-709a11b61adc</SharedGUID>
-  </PropertyGroup>
-  <PropertyGroup Label="Configuration">
-    <Import_RootNamespace>Microsoft.PowerFx.Connectors.Tests.Shared</Import_RootNamespace>
-  </PropertyGroup>
-  <ItemGroup>
-    <Compile Include="$(MSBuildThisFileDirectory)*.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Helpers\*.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <EmbeddedResource Include="$(MSBuildThisFileDirectory)Responses\*.eml" />
-    <EmbeddedResource Include="$(MSBuildThisFileDirectory)Responses\*.json" />
-    <EmbeddedResource Include="$(MSBuildThisFileDirectory)Responses\*.jsonSet" />
-    <EmbeddedResource Include="$(MSBuildThisFileDirectory)Responses\*.png" />
-    <EmbeddedResource Include="$(MSBuildThisFileDirectory)Responses\*.txt" />
-    <EmbeddedResource Include="$(MSBuildThisFileDirectory)Responses\*.raw" />
-    <EmbeddedResource Include="$(MSBuildThisFileDirectory)Swagger\*.json" />
-  </ItemGroup>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MSBuildAllProjects Condition="'$(MSBuildVersion)' == '' Or '$(MSBuildVersion)' &lt; '16.0'">$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <HasSharedItems>true</HasSharedItems>
+    <SharedGUID>b91ccc91-2a8d-4411-9a50-709a11b61adc</SharedGUID>
+  </PropertyGroup>
+  <PropertyGroup Label="Configuration">
+    <Import_RootNamespace>Microsoft.PowerFx.Connectors.Tests.Shared</Import_RootNamespace>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="$(MSBuildThisFileDirectory)AssemblyProperties2.cs" />

```

Comment:
>AssemblyProperties2 [](http://example.com/codeflow?start=49&length=19)

Is there a way to avoid explicitly listing every file? 
Like include *? 
This is going to trip folks up

Reply:
yes ... and no

Initially I used "*.cs"
This technically works but you cannot add a file in VS without corrupting the .projitems file.. 
After you add a file, VS will remove the "*.cs" reference and only keep the new file
So the only solution is to have all files there